src/modules/tests/test-data/cat-a-mod2/safety-and-balance/__tests__/safety-and-balance.cat-a-mod2.selector.spec.ts

```diff
@@ -0,0 +1,66 @@
+import { TestData } from '@dvsa/mes-test-schema/categories/AM2';
+import { safetyAndBalanceQuestionsExist } from '../safety-and-balance.cat-a-mod2.selector';
+
+describe('Vehicle Checks Selector Cat A Mod 2', () => {
+
+  describe('vehicleChecksExist', () => {
+    it('should return false if there are no vehicle checks entered', ()  => {
+      const emptySafetyAndBalanceQuestions: TestData['safetyAndBalanceQuestions'] = {
+        safetyQuestions: [{}, {}],
+        balanceQuestions: [{}],
+      };
+      const result = safetyAndBalanceQuestionsExist(emptySafetyAndBalanceQuestions);
+      expect(result).toBeFalsy();
+    });
+
+    it('should return true if there are safety questions entered', ()  => {
+      const populatedSafetyAndBalanceQuestions: TestData['safetyAndBalanceQuestions'] =
+        {
+          safetyQuestions: [{
+            code: 'SQ1',
+            description: 'test SQ1',
+            outcome: 'P',
+          }],
+          balanceQuestions: [{}],
+        };
+      const result = safetyAndBalanceQuestionsExist(populatedSafetyAndBalanceQuestions);
+      expect(result).toBeTruthy();
+    });
+
+    it('should return true if there are balance questions entered', ()  => {
+      const populatedSafetyAndBalanceQuestions: TestData['safetyAndBalanceQuestions'] =
+        {
+          safetyQuestions: [{}],
+          balanceQuestions: [{
+            code: 'BQ1',
+            description: 'test BQ1',
+            outcome: 'DF',
+          }],
+        };
+      const result = safetyAndBalanceQuestionsExist(populatedSafetyAndBalanceQuestions);
+      expect(result).toBeTruthy();
```

Comment:
I would always avoid using toBeTruthy and toBeFalsy and work prefer .toBe(true) .or toEqual(true) .

Those expect statements don't actually check for true and false and can lead to false positives. This blog explains more - https://vincenttunru.com/toBeTruthy-vs-toBe-true/